index.html

```diff
@@ -8,7 +8,6 @@
 </head>
 
 <body>
-
```

Comment:
-please be carful with blank lines and indentation only use them for readability purpose